app/models/exercise/practiced_concept.rb

```diff
@@ -0,0 +1,6 @@
+class Exercise::PracticedConcept < ApplicationRecord
```

Comment:
Let me know what you think of the class name.

Reply:
Yep - Like it!